chart/compass/values.yaml

```diff
@@ -23,7 +23,7 @@ global:
       version: "PR-1460"
     director:
       dir:
-      version: "PR-1504"
+      version: "PR-1529"
```

Comment:
There are some other resolvers involved in pairing such as applications, application templates. 
Also you've added some logs to subresolvers such as  api and event definitions but not to others like documents, fetch requests, package instance auths ,etc.